x-pack/plugins/observability_solution/apm/public/components/app/service_inventory/multi_signal_inventory/table/title_with_explanation.tsx

```diff
@@ -0,0 +1,162 @@
+                defaultMessage="Formula Calculation:"
+              />
+            </strong>
+            <br />
+            <code
+              css={css`
+                word-break: break-word;
+              `}
+            >
+              {formula}
+            </code>
+          </p>
+        </EuiText>
+      </EuiPopover>
+    </>
+  );
+}
+
+interface ExplanationProps {
+  includeTitle: boolean;
+}
+
+export function LogRateWithExplanation({ includeTitle }: ExplanationProps) {
+  return (
+    <TitleWithPopoverExplanation
+      name={
+        includeTitle
+          ? i18n.translate('xpack.apm.multiSignal.servicesTable.logRatePerMinute', {
+              defaultMessage: 'Log rate (per min.)',
+            })
+          : undefined
+      }
+      description={
+        <FormattedMessage
+          defaultMessage="Rate of logs per minute observed for given {serviceName}."
+          id="xpack.apm.multiSignal.servicesTable.logRatePerMinute.tooltip.description"
+          values={{
+            serviceName: (
+              <code
+                css={css`
+                  word-break: break-word;
+                `}
+              >
+                {i18n.translate(
+                  'xpack.apm.multiSignal.servicesTable.logRatePerMinute.tooltip.serviceNameLabel',
+                  {
+                    defaultMessage: 'service.name',
+                  }
+                )}
+              </code>
+            ),
+          }}
+        />
+      }
+      formula="count(kql='log.level: * OR error.log.level: *')"
+      data-test-subj={
+        includeTitle ? 'apmMultiSignalLogRateColumnTooltip' : 'apmMultiSignalLogRateChartTooltip'
+      }
+    />
+  );
+}
+
+export function LogErrorRateWithExplanation({ includeTitle }: ExplanationProps) {
+  return (
+    <TitleWithPopoverExplanation
+      name={
+        includeTitle
+          ? i18n.translate('xpack.apm.multiSignal.servicesTable.logErrorRate', {
+              defaultMessage: 'Log error rate',
+            })
+          : undefined
+      }
+      description={
+        <FormattedMessage
+          defaultMessage="% of logs where error detected for given {serviceName}."
+          id="xpack.apm.multiSignal.servicesTable.logErrorRate.tooltip.description"
+          values={{
+            serviceName: (
+              <code
+                css={css`
+                  word-break: break-word;
+                `}
+              >
+                {i18n.translate(
+                  'xpack.apm.multiSignal.servicesTable.logErrorRate.tooltip.serviceNameLabel',
+                  {
+                    defaultMessage: 'service.name',
+                  }
+                )}
+              </code>
+            ),
+          }}
+        />
+      }
+      formula={`count(kql='log.level: "error" OR log.level: "ERROR" OR  error.log.level: "error" OR error.log.level: "ERROR"') / count(kql='log.level: * OR error.log.level: *')`}
```

Comment:
Where can I find this formula? Is it in the transform? 

Moving forward, if this is applied to other metrics, how do we ensure the formulas here will reflect the actual ones that are used in the query? The formulas probably won't change often, so I'm not sure if this is something we need to be concerned about